### PR TITLE
docs: added spacebar brand guidelines

### PR DESCRIPTION
**Pull request by BRGustavoRibeiro**
Just to make it easier to find. We can improve it during the project with specific guidelines for buttons, font sizes and much more.

### CODE REVIEW

**Comment by BRGustavoRibeiro** [comment]
Is this PR good for merge? It would be nice to have this info over here

File: branding/README.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+Those are the guidelines for the Spacebar branding usage.
+
+## Colors
+- Primary: <img src="./colors/primary.svg" width="12px" height="12px"> #0185ff 
+- Secondary: <img src="./colors/secondary.svg" width="12px" height="12px">  #000115
+
+## Typography
+### Logo and Headings
+**New Hero (Adobe)**
```
Comment (Puyodead1) [inline branding/README.md:11]: maybe instead of the `(Adobe), just a link to the adobe font?

Comment (Puyodead1) [reply branding/README.md:11]: [New Hero](https://fonts.adobe.com/fonts/new-hero)

Comment (BRGustavoRibeiro) [reply branding/README.md]: Done! Also did it for Inter.

**Comment by Puyodead1** [comment]
lgtm

**Review by MaddyUnderStars — APPROVED** [review]

**Review by TheArcaneBrony — APPROVED** [review]
looks alright to me

**Pull request merged**